k8s/demo/common/ccd/ccd-demo-latest-chart.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: ccd-demo-latest-chart
+  namespace: ccd
+  annotations:
+    flux.weave.works/automated: 'true'
+    flux.weave.works/tag.java: 'glob:prod-*'
+spec:
+  releaseName: ccd-demo-latest-chart
+  timeout: 900
+  forceUpgrade: true
+  rollback:
+    enable: true
+    force: true
+  chart:
+    git: 'https://github.com/hmcts/chart-ccd'
+    ref: master
+    path: ccd
+  values:
+    ccd-api-gateway-web:
+      enabled: true
+      nodejs:
+        ingressHost: gateway-ccd-aks.service.core-compute-demo.internal
+    idam-pr:
+      releaseNameOverride: 'ccd-demo-idam-pr'
+      redirect_uris:
+        CCD:
+          - https://ccd-demo-case-management-web/oauth2redirect
```

Comment:
I don't know much about IDAm redirect urls, but doesnt this needs to be full url?

Reply:
can you please add a comment saying this is temporary until we add idam-pr to management web values.yml. We should push it there as a default or at least try, rather than adding it here or in ccd-chart.